### DISCUSSION TITLE
PR Readiness Feedback

### DISCUSSION THREAD

**Discussion by sheppardw** [TeamBasedInquiryLearning/precalculus#293] [General]
Link: [https://teambasedinquirylearning.github.io/precalculus/2024/PR.html](url)
Author: @kathypinzon
Reviewer:@sheppardw

**Comment by AbbyANoble** [comment]
Adding in some feedback for @kathypinzon so she can update before heading out of town early tomorrow!

Number 1 could also link to FN2.

**Comment by AbbyANoble** [comment]
Number 6 - extra parenthesis at the end of Khan Academy

**Comment by AbbyANoble** [comment]
Number 7 could also link to LF2

**Reply by AbbyANoble** [reply]
This would also be a good spot to link Check It once they are made.

**Comment by AbbyANoble** [comment]
Number 9 - With this readiness section already so long, this one may be a good spot to cut. I'm assuming it's in there for polynomial long division, but that might be better as a "look how these patterns are similar" activity in the section rather than in readiness.

**Reply by kathypinzon** [reply]
@siwelwerd @sheppardw this is already included as part of Activity 4.3.5 so I agree with Abby we don't really need it as part of readiness - or maybe we can leave it and just not assess on the RAT.

**Comment by AbbyANoble** [comment]
Number 10 - could also link to EQ5 and its CheckIt.

**Comment by AbbyANoble** [comment]
And if further trimming is needed, numbers 2 and 3 could go together potentially.

**Comment by AbbyANoble** [comment]
Number 4 is defined in [Definition 4.5.4](https://teambasedinquirylearning.github.io/precalculus/2024/instructor/PR5.html#degree-and-leading-coefficient) with an activity right after. May not need to be readiness if it's also an activity in the lesson.

**Reply by kathypinzon** [reply]
I am fine either way @tdegeorge that was your section do you want to keep the readiness or is the activity enough?
